src/services/doppler-billing-user-api-client.ts

```diff
@@ -137,6 +138,7 @@ export class HttpDopplerBillingUserApiClient implements DopplerBillingUserApiCli
       idConsumerType: data.idConsumerType,
       identificationType: data.identificationType,
       identificationNumber: data.identificationNumber,
+      responsableIVA: data.responsableIVA ? '1' : '0',
```

Comment:
Esta propiedad esta repetida en 
![image](https://user-images.githubusercontent.com/39057464/157931382-dc037151-16d2-4da5-96d7-875afbcb19d8.png)
 pero como boolean en lugar de string, ambos objetos se ven similares pero con diferencias menores en los nombres de las propiedades (y uno no esta tipado) quiza esto es correcto, pero me llamo la atencion.

Reply:
El boolean es el que se manda al BE y el otro es el mapeo al entidad PaymentMethod